src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -40,4 +40,29 @@ package object stage {
       }
     }
   }.isSuccess
+
+
+  /*
+   Logic opperators that work with Try[Boolean]
+   */
+  implicit class TryLogic(val t: Try[Boolean]) extends AnyVal {
```

Comment:
Also, it would be nice if you added the alternatives `Boolean 'operator' Try[Boolean]` and `Try[Boolean] 'operator' Boolean` back in.